lib/usd/ui/layerEditor/saveLayersDialog.cpp

```diff
@@ -165,26 +167,29 @@ SaveLayerPathRow::SaveLayerPathRow(
 
 QString SaveLayerPathRow::layerDisplayName() const { return _label->text(); }
 
-QString SaveLayerPathRow::absolutePath() const { return _absolutePath; }
-
-QString SaveLayerPathRow::pathToSaveAs() const { return _absolutePath; }
+QString SaveLayerPathRow::pathToSaveAs() const { return _pathToSaveAs; }
 
-void SaveLayerPathRow::setAbsolutePath(const std::string& path)
+void SaveLayerPathRow::setPathToSaveAs(const std::string& path)
 {
-    _absolutePath = path.c_str();
-    _pathEdit->setText(_absolutePath);
+    _pathToSaveAs = path.c_str();
+    _pathEdit->setText(_pathToSaveAs);
     _pathEdit->setEnabled(true);
 }
 
 void SaveLayerPathRow::onOpenBrowser()
 {
     std::string fileName;
     if (SaveLayersDialog::saveLayerFilePathUI(fileName)) {
-        setAbsolutePath(fileName);
+        if (MGlobal::optionVarExists(MAKE_PATH_RELATIVE_TO_SCENE_FILE)
+            && MGlobal::optionVarIntValue(MAKE_PATH_RELATIVE_TO_SCENE_FILE)) {
+            fileName = UsdMayaUtilFileSystem::getPathRelativeToMayaSceneFile(fileName);
```

Comment:
As suggested before, moving this into a common helper function would be preferable.